hugolib/page.go

```diff
@@ -330,13 +330,69 @@ func (p *Page) Param(key interface{}) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	keyStr = strings.ToLower(keyStr)
+	result, _ := p.traverseDirect(keyStr)
+	if result != nil {
+		return result, nil
+	}
+
+	keySegments := strings.Split(keyStr, ".")
+	if len(keySegments) == 1 {
+		return nil, nil
+	} else {
```

Comment:
OK, this is nit picky, but I guess the Go linter will complain about this: You should drop the else here, as you return early above.